### ISSUE TITLE
.env support file for slack tokens

### ISSUE THREAD

**Issue by peterhogg** [evanyeung/terminal-slack#54]

**Comment by grifball** [comment]
So, you want it to read from a file on startup to grab the token?

**Comment by peterhogg** [comment]
Correct. Either through the [dotenv](https://www.npmjs.com/package/dotenv) package as to keep the current functionality the same, or through a standard config file.
